README.md

```diff
@@ -3,30 +3,55 @@ Official rulebook for the Lords &amp; Lads stump/hammerschlagen variant
 
 ![Cover Art](/images/cover.png)
 
+# Table of Contents
+* I - [Materials Required](i---materials-required)
```

Comment:
I like the ToC idea and think it will make it much easier to look things up

Now that I'm seeing the sections laid out this way though, I'm thinking we may want to structure it to make the ToC.

For instance, the top level sections don't really feel like top level sections, especially since 90% of the content is under `The Game`

I'm thinking we could put `Materials Required` and `Setting up` before `Overview` as siblings then remove the `The Game` header altogether and un-indent everything by one.

I.E.

```
#Table of Contents
I. Materials Required
II Settings Up
III Overview
IV Taking a Turn
V Hammer Test
```
etc

Reply:
I agree, done